### ISSUE TITLE
Duplicate template name in boilerplate template docs

### ISSUE THREAD

**Issue by davidrea** [VeliovGroup/flow-router#38]
If the code and HTML provided in the [templating guide](https://github.com/VeliovGroup/flow-router/blob/master/docs/templating.md) are used verbatim, Meteor complains (in console) about duplicate templates named "index".

The "loading" template re-uses the template name "index":

```javascript
<!-- /imports/client/loading/loading.html -->
<template name="index">
  Loading...
</template>
```

**Comment by mikhail-shishov** [comment]
Hello @davidrea ,

Let us know more details regarding this issue, please. Make sure you follow our [issue template](https://github.com/VeliovGroup/flow-router/blob/master/.github/ISSUE_TEMPLATE) when describing the problem.

**Comment by davidrea** [comment]
From issue template: 

>### Documentation is missing something or incorrect (have typos, etc.):
>  - Give an expressive description what you have changed/added and why
>  - Make sure you're using correct markdown markup
>  - Make sure all code blocks starts with tripple ``` (*backtick*) and have a syntax tag, for more read [this docs](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting)
>  - Post your addition/changes in issue, we will manage it

Expressive description: ✔
Markdown markup correct: ✔
Code blocks triple-backticked with syntax tag: ✔
Change posted in issue: ✔

All that needs to be changed is `<template name="index">` to `<template name="loading">` in flow-router/docs/templating.md in the "Create Loading Template" section. See pull request.

**Comment by dr-dimitru** [comment]
@davidrea thank you for sending a PR, and spotting this dummy mistake.
@mksh-su this is documentation erratum, solved with #39 PR.
